dockerfiles/clojure-1.12.0.Dockerfile

```
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1.7-labs
+FROM clojure:tools-deps-bookworm
+
+# Ensures the container is re-built if dependency files change
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="deps.edn"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)